rollup/rollup.config.js

```diff
@@ -12,20 +12,19 @@ const buildDate = Intl.DateTimeFormat('en-CA', { timeZone: 'America/Toronto' }).
 const jsGlob = [
 	'@(components|controllers|directives|helpers|mixins|templates|test)/**/*.js',
 	'./index.js',
-	'!**/*.@(test|axe|visual-diff|vdiff).js',
+	'!**/*.@(test|axe|vdiff).js',
 ];
 const nonJsGlob = [
 	'@(components|controllers|directives|helpers|mixins|templates|test)/**/*.*',
 	'*.*',
 	'!**/*.@(js|md|json)',
-	'!**/screenshots/**/*',
 	'!**/golden/**/*',
 ];
 
 export default {
 	input: glob.sync(jsGlob),
 	output: { dir: 'build', format: 'es', preserveModules: true },
-	external: ['puppeteer', '@brightspace-ui/visual-diff', '@brightspace-ui/testing', 'sinon'],
+	external: ['puppeteer', '@brightspace-ui/testing', 'sinon'],
```

Comment:
Do I still need `puppeteer` here?

Reply:
Nothing should be using it anymore, so should be safe to remove.